ote_sdk/ote_sdk/usecases/exportable_code/prediction_to_annotation_converter.py

```diff
@@ -145,7 +150,7 @@ def create_converter(
 
     converter: IPredictionToAnnotationConverter
     if converter_type == Domain.DETECTION:
-        converter = DetectionBoxToAnnotationConverter(labels)
+        converter = DetectionToAnnotationConverter(labels)
```

Comment:
Can we remove "DetectionBoxToAnnotationConverter" then?

Reply:
yeah, I can, I will rerun the tests